legal-api/src/legal_api/reports/business_document.py

```diff
@@ -412,6 +422,8 @@ def _format_state_filing(self, filing: Filing) -> dict:
             continuation_out_date = LegislationDatetime.as_legislation_timezone_from_date_str(
                 filing_meta['continuationOut']['continuationOutDate'])
             filing_info['continuationOutDate'] = continuation_out_date.strftime(OUTPUT_DATE_FORMAT)
+        elif filing.filing_type == 'amalgamationApplication':
+            filing_info['filingName'] = 'Amalgamation'
```

Comment:
Confirm with Yui, this one should be `amalgamation`, instead of `amalgamated` in the previous `Mockups`